packages/medusa/src/api/routes/admin/products/index.ts

```diff
@@ -138,6 +138,95 @@ export const defaultAdminProductFields: (keyof Product)[] = [
 
 export const defaultAdminGetProductsVariantsFields = ["id", "product_id"]
 
+/**
+ * This is temporary.
+ */
+export const defaultAdminRemoteQueryFields = {
```

Comment:
suggestion(non-blocking): If we start using these elsewhere, it will be useful with a more descriptive name:
```suggestion
export const defaultAdminProductRemoteQueryObject = {
```

Reply:
I agree, although it is temporary. i am planning on instead updating the current default fields and build an util that shape it to a remote query object. Wdyt?

Reply:
@olivermrbl here is the [util](https://github.com/medusajs/medusa/pull/5161/files) which will allow us in the short term to get rid of that object in the index. I did that here only to not duplicate it everywhere at the moment

Reply:
Will have a look at the util!